### PR TITLE
Fabo/2202+2204 fix number

### LINKED ISSUE #2202: Very small amounts are displayed wrong
**Describe the bug**

image

**To Reproduce**

Go to live URL
Click on faucet
Receive `1cococoin`

**Expected behavior**

This should still show a number `0.0000001`

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by faboweb**
Closes #2202, #2204 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
fixed some changes from "CHANGELOG.md" to "PENDING.md"

File: app/src/renderer/components/transactions/TmLiStakeTransaction.vue [hunk 1]
```diff
@@ -69,7 +65,7 @@
 <script>
 import TmLiTransaction from "./TmLiTransaction"
 import colors from "./transaction-colors.js"
-import { pretty, atoms } from "../../scripts/num.js"
+import { pretty, full, atoms } from "../../scripts/num.js"
```
Comment (jbibla) [inline app/src/renderer/components/transactions/TmLiStakeTransaction.vue]: is `pretty` being used?

File: app/src/renderer/components/transactions/TmLiStakeTransaction.vue [hunk 2]
```diff
@@ -45,12 +45,8 @@
               calculatePrettifiedTokens(tx.validator_addr, tx.shares_amount)
             }}
           </b><span>&nbsp;{{ bondingDenom }}s</span>
-        </template><template
-          v-if="timeDiff"
-        >
-          <span
-            class="tx-unbonding__time-diff"
-          >
+        </template><template v-if="timeDiff">
```
Comment (jbibla) [inline app/src/renderer/components/transactions/TmLiStakeTransaction.vue]: don't love this spacing

File: app/src/renderer/scripts/num.js [hunk 3]
```diff
@@ -6,67 +6,39 @@ import BigNumber from "bignumber.js"
  * @module num
  */
 
-const numeral = require(`numeral`)
-function usd(num) {
-  return numeral(num).format(`$0,0.00`)
+const language = window.navigator.userLanguage || window.navigator.language
```
Comment (jbibla) [inline app/src/renderer/scripts/num.js]: interesting. what is this for?

**Pull request merged**